test/internals/reply.test.js

```diff
@@ -293,6 +293,38 @@ test('buffer with content type should not send application/octet-stream', t => {
   })
 })
 
+test('stream using reply.res.writeHead should return customize headers', t => {
+  t.plan(4)
+
+  const fastify = require('../..')()
+  const fs = require('fs')
+  const path = require('path')
+
+  let streamPath = path.join(process.cwd(), './package.json')
```

Comment:
Can you use path.join(__dirname, ‘..’, ‘package.json’) instead? it’s more robust.

Reply:
Can you also use `var` instead of `let`?

Reply:
General speaking `let` is slower, but in this case for consistency across the test :)

Reply:
@delvedor OK, I got it :)